plugins/main/bridge/bridge.go

```diff
@@ -96,19 +103,23 @@ func ensureBridgeAddr(br *netlink.Bridge, ipn *net.IPNet, forceAddress bool) err
 	return nil
 }
 
-func deleteBridgeAddr(br *netlink.Bridge, ipn *net.IPNet) error {
+func deleteBridgeAddr(br *netlink.Bridge, ipn *net.IPNet, isIPV4 bool) error {
 	addr := &netlink.Addr{IPNet: ipn, Label: ""}
 
-	if err := netlink.LinkSetDown(br); err != nil {
-		return fmt.Errorf("could not set down bridge %q: %v", br.Name, err)
+	if isIPV4 {
+		if err := netlink.LinkSetDown(br); err != nil {
+			return fmt.Errorf("could not set down bridge %q: %v", br.Name, err)
+		}
```

Comment:
I was wondering the same thing, and was hoping someone had some historical context for why this the link flapping is there. I'll remove this in next update.